.gitignore

```diff
@@ -53,6 +54,6 @@ src/test/struct
 src/test/struct_migrate
 src/test/zboxloop
 src/examples/ex[0-9]
```

Comment:
Please also ignore the generated .exe files for the Windows platform.

Reply:
Done, thanks!